setup.py

```diff
@@ -4,15 +4,14 @@
 
 from setuptools import setup, find_packages
 
-REQUIRED_PACKAGES = [
-    'gym',
-    'tensorflow-probability',
-]
+REQUIRED_PACKAGES = ["gym"]
```

Comment:
Why is `tensorflow` not a requirement?

Reply:
When you `pip install` the package you don't want to assume what kind of TensorFlow the user wants to install. This is the same pattern followed by libraries like `tensorboard` and `tensorflow_probability` so it remains compatible with CPU/GPU variants as well as not requiring an exact version match.

The `Pipfile` is just for local installation and development so it specifies the full dependencies list.